base_wamas_ubl/lib/wamas/tests/test_wamas2ubl.py

```diff
@@ -34,6 +34,11 @@ def test_convert_wamas2ubl_reception(self):
         lst_expected_output = "WAMAS2UBL-SAMPLE_WEAKQ_WEAPQ-DESPATCH_ADVICE.xml"
         self._convert_wamas2ubl(input_file, lst_expected_output)
 
+    def test_convert_wamas2ubl_reception2(self):
```

Comment:
```suggestion
    def test_convert_wamas2ubl_reception_combined(self):
    """Test a reception with a single header concerning multiple receptions
    
    This happens when a supplier deliver goods for multiple INs"""
```

And rename `test_convert_wamas2ubl_reception` into `test_convert_wamas2ubl_reception_simple` so that you don't have a test which is a prefix of another test